src/sentry/api/endpoints/group_tags.py

```diff
@@ -37,4 +38,20 @@ def get(self, request: Request, group: Group) -> Response:
             group, environment_ids, keys=keys, value_limit=value_limit
         )
 
-        return Response(serialize(tag_keys, request.user))
+        data = serialize(tag_keys, request.user)
+
+        show_readable_tag_values = request.GET.get("readable")
+        if show_readable_tag_values:
+            add_readable_tag_values(data)
+
+        return Response(data)
+
+
+def add_readable_tag_values(data: Any) -> Any:
+    # Map device tag to a more readable value if possible
+    device_tag = next((tag for tag in data if tag["key"] == "device"), None)
```

Comment:
nit: Could this just be a for loop with a break instead?

Reply:
Updated to a for loop!